Add December events to repository

laisbsc opened the pull request:
Draft of the blogpost for Event I of the JR & WhyR partnership.

I've changed the format a bit from the 'template' and really like how it is looking. Let me know if you'd like to change things, especially regarding formatting.

laisbsc committed:
'claus_anne_reporteR'

laisbsc committed:
'delete_file'

MarcinKosinski commented:
Hey @laisbsc this looks good! I wouldn't delete the `/foundation/images/fulls/webinars/reporter.jpg` file and I would keep it with the name as it is becuase it is also used here in line 35 https://github.com/WhyRFoundation/foundation/blob/master/webinars.md
Can you change the name back to reporteR (or at least something short) and make sure it's consistenr with the list of webinars in webinars.md ?

One extra favour - would you mind updating Claus and Anne setion in webinars.md with their names+urls and urls to meetup and youtube?

MarcinKosinski reviewed (changes requested):
As on the comment that I left on PR

laisbsc commented:
> 
> 
> Hey @laisbsc this looks good! I wouldn't delete the `/foundation/images/fulls/webinars/reporter.jpg` file and I would keep it with the name as it is because it is also used here in line 35 https://github.com/WhyRFoundation/foundation/blob/master/webinars.md

Sure. 

> Can you change the name back to reporteR (or at least something short) and make sure it's consistenr with the list of webinars in webinars.md ?

No problem, back to `reporteR.jpg`.

> One extra favour - would you mind updating Claus and Anne setion in webinars.md with their names+urls and urls to meetup and youtube? 

So, apart from writing the blog post and all the social media and setting up the events online, I need to keep a public summary on the `webinar.md` file. Is that right?
Can I ask you to add such action to the onboarding document, please? I am still not sure I will remember all the steps for this webinars 😞

MarcinKosinski commented:
@laisbsc can we also get those 3 bullet points somewhere in the blog post (preferrably at the beginning)

- webinars [whyr.pl/webinars/](http://whyr.pl/webinars/)
- donate [whyr.pl/donate/](http://whyr.pl/donate/)
- subscribe [whyr.pl/donate/](http://whyr.pl/subscribe/)
- past videos [youtube.com/WhyRFoundationVideos](https://www.youtube.com/WhyRFoundationVideos)

So that people have a link to all webinars, can potentially subscribe to the newsletter and may someday donate (we once got 4pounds! : D )

MarcinKosinski commented:
Sure @laisbsc let me update. Sorry for that!

laisbsc commented:
> 
> 
> @laisbsc can we also get those 3 bullet points somewhere in the blog post (preferrably at the beginning)
> 
>     * webinars [whyr.pl/webinars/](http://whyr.pl/webinars/)
> 
>     * donate [whyr.pl/donate/](http://whyr.pl/donate/)
> 
>     * subscribe [whyr.pl/donate/](http://whyr.pl/subscribe/)
> 
>     * past videos [youtube.com/WhyRFoundationVideos](https://www.youtube.com/WhyRFoundationVideos)
> 
> 
> So that people have a link to all webinars, can potentially subscribe to the newsletter and may someday donate (we once got 4pounds! : D )

If you don't mind, I prefer not to add links like this. I am happy to embed them on text and give some context but I really don't like to see extensive links on blog posts when I am reading them.
Let me see what I can do.

And I am really sorry but I will only have time to take care of this in the evening. In class today and hosting events and organising the conference and 2 assignments due for the next 3 days.

And thanks a million for the quick and awesome feedback.

laisbsc commented:
> 
> 
> Sure @laisbsc let me update. Sorry for that!

No need to apologise at all! It is our first event setup together, we're adapting and finding the balance :)

Thanks for all the help and support <3

MarcinKosinski commented:
@laisbsc take your time. No worries. We still have plenty of time, so what if we include link to the youtube channel and to the subsrcibe page in one sentence at the end?

MarcinKosinski commented:
And then I think you'll need to update this PR since changes are not visible on this thread

laisbsc committed:
'fix_blog_posts'

laisbsc commented:
> 
> 
> @laisbsc take your time. No worries. We still have plenty of time, so what if we include the link to the youtube channel and to the subscribe page in one sentence at the end?

I've changed a few things there and added a nice request for donations. As always, you're more than welcome to just change things, I ain't precious and I know how much effort you've been putting into those 😄 

Hope you like it! 😘

MarcinKosinski commented:
Will give it a look in a second! Thanks!

On Thu, 26 Nov 2020, 17:27 Lais Carvalho, <notifications@github.com> wrote:

> @laisbsc <https://github.com/laisbsc> take your time. No worries. We
> still have plenty of time, so what if we include the link to the youtube
> channel and to the subscribe page in one sentence at the end?
>
> I've changed a few things there and added a nice request for donations. As
> always, you're more than welcome to just change things, I ain't precious
> and I know how much effort you've been putting into those 😄
>
> Hope you like it! 😘
>
> —
> You are receiving this because your review was requested.
> Reply to this email directly, view it on GitHub
> <https://github.com/WhyRFoundation/foundation/pull/1#issuecomment-734391994>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABTVVTX22TYPN35PNFPU3H3SRZ6XJANCNFSM4UCJFLAA>
> .
>

MarcinKosinski committed:
typo

MarcinKosinski reviewed (approved):
This is solid! I just fixed 1 typo but this is minor

MarcinKosinski commented:
@laisbsc 
> So, apart from writing the blog post and all the social media and setting up the events online, I need to keep a public summary on the webinar.md file. Is that right?
Can I ask you to add such action to the onboarding document, please? I am still not sure I will remember all the steps for this webinars 😞

Just added 👇 
https://docs.google.com/document/d/1OvJ2Drijhjaj-Rk51ZEaHd6lapv8Zo1ZJ3FCyWwTqU8/edit?disco=AAAAKryYh68
